src/spec/mpq.spec.js

```diff
@@ -94,4 +94,78 @@ describe('MPQ', () => {
       mpq.close();
     });
   });
+
+  describe('Search', () => {
+    test('finds all files', async () => {
+      const mpq = await MPQ.open('/fixture/vanilla-standard.mpq');
+
+      const results = mpq.find('*');
+
+      expect(results).toBeInstanceOf(Array);
+
+      expect(results.map((r) => r.fileName)).toEqual([
+        'fixture-deDE.txt',
+        '(listfile)',
+        'nested\\fixture-nested.txt',
+        'fixture.png',
+        '(attributes)',
+        'fixture.txt',
+        'fixture.xml'
+      ]);
+
+      mpq.close();
+    });
+
+    test('returns result with appropriate shape', async () => {
+      const mpq = await MPQ.open('/fixture/vanilla-standard.mpq');
+
+      const result = mpq.find('fixture.txt')[0];
+
+      expect(result).toEqual({
+        'fileName': 'fixture.txt',
+        'plainName': 'fixture.txt',
+        'hashIndex': 3886,
+        'blockIndex': 0,
+        'fileSize': 13,
+        'compSize': 21,
+        'fileTimeLo': 414638976,
+        'fileTimeHi': 30643794,
+        'locale': 0
```

Comment:
Any specific reason for these properties being quoted?

Reply:
Copy pasta! I brought the shape over from `EmStormLib.cpp`. I'll fix that. :)